intel-sgx/aesm-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aesm-client"
-version = "0.5.4"
```

Comment:
Hi @nshyrei, please check https://semver.org/
The version convention is `MAJOR.MINOR.PATCH`. Bump the PATCH version does not represent the breaking change.

You need to at least bump the MINOR version.
This applies to all your other version update, too.

Could you fix them, thank you.

Reply:
You could use https://crates.io/crates/cargo-semver-checks to check if the version is file after you update them.